anndata/_core/aligned_mapping.py

```diff
@@ -243,7 +246,10 @@ def __init__(
         self._parent = parent_view
         self.subset_idx = subset_idx
         self._axis = parent_mapping._axis
-        self.dim_names = parent_mapping.dim_names[subset_idx]
+
+    @property
+    def dim_names(self) -> pd.Index:
+        return self.parent_mapping.dim_names[self.subset_idx]
```

Comment:
Shouldn't this just be the same as for the `Actual` class?

Reply:
So this could just be a property on the parent class

Reply:
I am a bit confused, is there already a dim_names in AxisArraysBase: 

https://github.com/theislab/anndata/pull/461/files/bac91c25d502ed72daa499b07b08d198302acb3e#diff-af8b8ed3c983f1ed77b81030eb9d80df8136e0640324f180354e6e0d3b49e7e7R198

Reply:
Here I mean:

https://github.com/theislab/anndata/blob/master/anndata/_core/aligned_mapping.py#L198

If yes, where does it come from?

Reply:
Are you asking how ` AxisArraysBase` can refer to the `dim_names` attribute? If so:

The idea is that there will never actually be an instance of `AxisArraysBase`. The class is only meant to be inherited from so `AxisArrays` and `AxisArraysView` can share code.

Reply:
Hmm I see. But then things like `_axis` and `dim_names` must be a "requirement" in the AlignedMapping class, no?

Reply:
What I mean is: if I were to write another python package where I define classes inheriting from `AxisArraysBase`, I wouldn't know that I have to have `_axis` and `dim_names` in my class.

Reply:
True, but we don't export `AxisArraysBase`, so it's a bit use-at-your-own-risk. For now, I'd call these classes strictly internal.